Components/Hero.js

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from 'react';
+
+import '../css/Components/Hero'
+
+import Title from './Title';
+import SubSectionTitle from './SubSectionTitle';
+
+import PlayIcon from '../images/play.svg';
+import TwitterLogo from '../images/twitter.svg';
+import YoutubeLogo from '../images/youtube.svg';
+import FacebookLogo from '../images/facebook.svg';
+
+export default class Hero extends Component {
+  render() {
+    return <section className="Hero">
+      <div className="Hero-background">
+      </div>
+
+      <div className="Hero-foreground">
+        <div className="Hero-content">
+          <div className="Hero-title">
+            <SubSectionTitle>Mirror Conference 2.0</SubSectionTitle>
```

Comment:
Why are these called `SubSectionTitle` when the main one is just called `Title`? This makes the assumption that they'll only be used for sections, which I'm not sure is true, and seems dangerous nonetheless. The styleguide doesn't mention this dinstinction either

How about just calling them `SubTitle`?